nilearn/decoding/tests/test_decoder.py

```diff
@@ -308,6 +361,66 @@ def test_parallel_fit(rand_X_Y):
             assert a == b
 
 
+@pytest.mark.parametrize(
+    "estimator, param_name, fitted_param_name, is_classification",
+    [
+        (RidgeCV(), "alphas", "best_alpha", False),
+        (RidgeClassifierCV(), "alphas", "best_alpha", True),
```

Comment:
I may be missing something, but what was the reason for adding `param_name`, `fitted_param_name` to the parametrization given that they are the same for both cases?

Reply:
gotcha: 👍🏾 
thanks!